korlibs-template/test/korlibs/template/TemplateTest.kt

```diff
@@ -51,6 +51,22 @@ class TemplateTest : BaseTest() {
         assertEquals("soywizsoywiz", KorteTemplate("{{name}}{{ name }}")("name" to "soywiz"))
     }
 
+    @Test
+    fun testSimpleIs() = suspendTest {
```

Comment:
This test should be moved here, so it is only executed on the JVM: `https://github.com/korlibs/korlibs/blob/main/korlibs-template/test%40jvm/korlibs/template/TemplateJvmTest.kt`